apps/web/src/app/daily/questions/[questionId]/_components/importance-evaluation.tsx

```
@@ -0,0 +1,51 @@
+"use client";
+
+import * as React from "react";
+import { Button } from "@/components/button/button";
+import { StarRating } from "@/components/star/star-rating";
+import { cn } from "@/lib/cn";
+
+interface ImportanceEvaluationProps {
+  open?: boolean;
+}
+
+function ImportanceEvaluation({}: ImportanceEvaluationProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)